lib/jekyll/drops/drop.rb

```diff
@@ -209,6 +209,18 @@ def fetch(key, default = nil, &block)
         return yield(key) unless block.nil?
         return default unless default.nil?
       end
+
+      private
+      def marshal_dump
+        (instance_variables - [:@context]).map do |attr|
+          { attr => instance_variable_get(attr) }
+        end.inject(:merge)
```

Comment:
`array.inject(:merge)` was known to be *very slow..!* (Not sure if it still is..) in comparison to `array.inject(:merge!)` and `array.each_with_object` (..though not sure that `#each_with_object` defines the `default_proc` for the *internal memo*)

Reply:
Good catch, thanks! 👍  I just ran a quick benchmark and `inject(:merge)` is indeed much, much slower than `injcect(:merge!)`. Will update

Reply:
it would be nice to have that `inject-merge-vs-inject-merge!` benchmark added to *our* Benchmark Suite.. (perhaps better in a separate PR)